deb/common/rules

```diff
@@ -10,8 +10,7 @@ override_dh_auto_build:
 	cd engine && TMP_GOPATH="/go" hack/dockerfile/install/install.sh proxy dynamic
 	cd engine && TMP_GOPATH="/go" hack/dockerfile/install/install.sh rootlesskit dynamic
 	# Build the CLI
-	cd /go/src/github.com/docker/cli && \
-		LDFLAGS='' DISABLE_WARN_OUTSIDE_CONTAINER=1 make VERSION=$(VERSION) GITCOMMIT=$(CLI_GITCOMMIT) dynbinary manpages
```

Comment:
Why did you remove the LDFLAGS='' here?

(perhaps we need the same comment as in the plugins install script?

Reply:
I didn't think enough, and had too much confidence in Tonis's scripts :) I put it back.